exporter/datadogexporter/internal/utils/tags.go

```diff
@@ -0,0 +1,29 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"fmt"
+)
+
+// FormatKeyValueTag takes a key-value pair, and creates a tag string out of it
+// Tags can't end with ":" so we replace empty values with "n/a"
+func FormatKeyValueTag(key, value string) string {
```

Comment:
This needs to live inside the `translator` package since we want to eventually move this code to the datadog-agent repository

Reply:
I put the function in a new `internal/translator/utils` package, otherwise there would be circular import errors:
```
package github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datadogexporter
        imports github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datadogexporter/internal/translator
        imports github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datadogexporter/internal/instrumentationlibrary
        imports github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datadogexporter/internal/translator: import cycle not allowed
```

The other solution would be to move the `instrumentationlibrary` package inside `translator`, but it's less clean (and we didn't do this for the similar `TagsToAttributes` function, which is in the `attributes` package).

Reply:
Makes sense doing it this way